app/build.gradle

```diff
@@ -121,6 +124,11 @@ android {
         }
     }
     namespace 'at.tomtasche.reader'
+    compileSdk 35
+    buildToolsVersion '34.0.0'
+
+    // TODO can and should this be architecture dependent?
+    sourceSets.main.assets.srcDirs += "build/conan/armv8/assets"
```

Comment:
@TomTasche not sure if this is even relevant

Reply:
I guess the assets we care about are not arch-dependent? still makes sense to keep the TODO in case someone considers adding arch-dependent assets...